src/Core/ServiceWrapper/Main.cs

```diff
@@ -356,41 +348,40 @@ private void WaitForProcessToExit(Process processoWait)
 
             try
             {
-//                WriteEvent("WaitForProcessToExit [start]");
+                // WriteEvent("WaitForProcessToExit [start]");
 
                 while (!processoWait.WaitForExit(effectiveProcessWaitSleepTime))
                 {
                     SignalShutdownPending();
-//                    WriteEvent("WaitForProcessToExit [repeat]");
+                    // WriteEvent("WaitForProcessToExit [repeat]");
                 }
             }
             catch (InvalidOperationException)
             {
                 // already terminated
             }
 
-//            WriteEvent("WaitForProcessToExit [finished]");
+            // WriteEvent("WaitForProcessToExit [finished]");
         }
 
         private void SignalShutdownPending()
         {
             int effectiveWaitHint;
-            if (_descriptor.WaitHint.TotalMilliseconds > Int32.MaxValue)

```

Comment:
IMHO explicit type is better for readability, especially for those ones who are not familiar with .NET type standards

Reply:
There are some additional considerations:

https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html